cosmic-core/api/src/main/java/com/cloud/api/command/admin/cloudops/ListWhoHasThisMacCmd.java

```diff
@@ -0,0 +1,41 @@
+package com.cloud.api.command.admin.cloudops;
+
+import com.cloud.api.APICommand;
+import com.cloud.api.ApiConstants;
+import com.cloud.api.BaseListDomainResourcesCmd;
+import com.cloud.api.Parameter;
+import com.cloud.api.response.ListResponse;
+import com.cloud.api.response.WhoHasThisIpResponse;
+
+@APICommand(name = "listWhoHasThisMac", description = "Lists all for this IP address", responseObject = WhoHasThisIpResponse.class)
```

Comment:
I'd suggest to have an abstract API response that `WhoHasThisIpResponse` and `WhoHasThisMacResponse` would extend from.